pytorch_blade/torch_blade/pass_manager.py

```diff
@@ -121,7 +121,7 @@ def _jit_pass_lower_to_onnx(graph):
     #
     # But something weird happened:
     # call to ConstantValueMap::ClearMaps() in C++ will segfault
-    if utils.torch_version_number() >= utils.parse_version("1.11.0"):
+    if utils.torch_version_number() >= utils.parse_version("1.9.0"):
```

Comment:
What's special about this version?

Reply:
The API of some ONNX passes has been changed since 1.9.0 but not 1.11.0.